spec/main_spec.rb

```diff
@@ -0,0 +1,7 @@
+require './lib/board.rb'
+require './bin/main.rb'
+
+describe User do
+  let(:game) { Board.new }
+  let(:board) { game.init_board(9) }
+end
```

Comment:
I can not see any test cases. Where are the test cases? Well, I checked your main.rb and the class User seems to contain tightly coupled methods. Check out this [article](https://www.codewithjason.com/test-ruby-methods-involve-puts-gets/), and let's find some cases from there that you can add in here.